src/components/shared/menu.js

```diff
@@ -68,7 +72,7 @@ function showMenu(e, items) {
   }
 
   menu.on("open", (_, popup) => onShown(menu, popup));
-  return menu.popup(e.clientX, e.clientY, { doc: document });
+  menu.popup(e.clientX, e.clientY, { doc: document });
```

Comment:
👍 

Yep, returning the popup element could allow some dangerous things to happen